### ISSUE TITLE
unable to export clipped file

### ISSUE THREAD

**Issue by PinkDucki** [mifi/lossless-cut#1351]
### I have a lot of issues to go through, so in order to make it easier for me to help you, I ask that you please try these things first

- [X] Try with the [newest version from GitHub](https://github.com/mifi/lossless-cut/releases/latest)
- [X] Read the [documentation](https://github.com/mifi/lossless-cut) and [FAQ, Known issues, Troubleshooting](https://github.com/mifi/lossless-cut/blob/master/issues.md)
- [X] Search for your problem under [Issues](https://github.com/mifi/lossless-cut/issues) or [Discussions](https://github.com/mifi/lossless-cut/discussions)
- [X] If you have an issue only with one particular file, [use this form instead](https://github.com/mifi/lossless-cut/issues/new?assignees=&labels=&template=file-issue.yml)

### Operating System

Windows 10

### Steps to reproduce

I am not sure what the issue is so I cant say how to reproduce it, but I can explain what is wrong. 

When I try to export a clipped video, this message pops up:
![image](https://user-images.githubusercontent.com/117027914/198886820-15884e68-4b45-4396-8809-d41d665a814e.png)

I have tried all of the steps provided in that pop up, and even tried uninstalling and reinstalling losslesscut (both in the same location and on another drive), but even after all of that, the message kept popping up, so i pressed the report button and came here. After pressing the report button, there is a section that says 'Include the following text'. Here is that text: 

Error: Command failed with exit code 1: G:\LosslessCut\resources\ffmpeg.exe -hide_banner -ss 459.15498 -i E:\Videos\R6 Clips\Desktop\Desktop 2022.09.29 - 01.06.28.02.DVR.mp4 -t 18.66746 -avoid_negative_ts make_zero -map 0:0 -c:0 copy -map 0:1 -c:1 copy -map_metadata 0 -movflags +faststart -default_mode infer_no_subs -ignore_unknown -f mp4 -y E:\Videos\Destiny 2 - edits\yes
Input #0, mov,mp4,m4a,3gp,3g2,mj2, from 'E:\Videos\R6 Clips\Desktop\Desktop 2022.09.29 - 01.06.28.02.DVR.mp4':
  Metadata:
    major_brand     : mp42
    minor_version   : 0
    compatible_brands: isommp42
    creation_time   : 2022-09-28T15:07:43.000000Z
    date            : 2022
  Duration: 00:08:00.13, start: 0.000000, bitrate: 40494 kb/s
  Stream #0:0[0x1](und): Video: h264 (High) (avc1 / 0x31637661), yuv420p(tv, smpte170m, progressive), 1920x1080 [SAR 1:1 DAR 16:9], 40292 kb/s, 59.87 fps, 60 tbr, 90k tbn (default)
    Metadata:
      creation_time   : 2022-09-28T15:07:43.000000Z
      handler_name    : VideoHandle
      vendor_id       : [0][0][0][0]
  Stream #0:1[0x2](und): Audio: aac (LC) (mp4a / 0x6134706D), 48000 Hz, stereo, fltp, 195 kb/s (default)
    Metadata:
      creation_time   : 2022-09-28T15:07:43.000000Z
      handler_name    : SoundHandle
      vendor_id       : [0][0][0][0]
E:\Videos\Destiny 2 - edits\Soul eager me what did i die to?: Invalid argument
    at makeError (G:\LosslessCut\resources\app.asar\node_modules\execa\lib\error.js:60:11)
    at handlePromise (G:\LosslessCut\resources\app.asar\node_modules\execa\index.js:118:26)
    at async file:///G:/LosslessCut/resources/app.asar/build/static/js/main.ff7aecec.js:2:307871
    at async file:///G:/LosslessCut/resources/app.asar/build/static/js/main.ff7aecec.js:2:308737
    at async file:///G:/LosslessCut/resources/app.asar/build/static/js/main.ff7aecec.js:2:2386227

{
  "err": {
    "killed": false,
    "failed": true,
    "timedOut": false,
    "isCanceled": false,
    "exitCode": 1
  },
  "state": {
    "filePath": "E:\\Videos\\R6 Clips\\Desktop\\Desktop 2022.09.29 - 01.06.28.02.DVR.mp4",
    "fileFormat": "mp4",
    "externalFilesMeta": {},
    "mainStreams": [
      {
        "index": 0,
        "codec_name": "h264",
        "codec_long_name": "H.264 / AVC / MPEG-4 AVC / MPEG-4 part 10",
        "profile": "High",
        "codec_type": "video",
        "codec_tag_string": "avc1",
        "codec_tag": "0x31637661",
        "width": 1920,
        "height": 1080,
        "coded_width": 1920,
        "coded_height": 1080,
        "closed_captions": 0,
        "film_grain": 0,
        "has_b_frames": 0,
        "sample_aspect_ratio": "1:1",
        "display_aspect_ratio": "16:9",
        "pix_fmt": "yuv420p",
        "level": 42,
        "color_range": "tv",
        "color_space": "smpte170m",
        "color_transfer": "smpte170m",
        "color_primaries": "smpte170m",
        "chroma_location": "left",
        "field_order": "progressive",
        "refs": 1,
        "is_avc": "true",
        "nal_length_size": "4",
        "id": "0x1",
        "r_frame_rate": "60/1",
        "avg_frame_rate": "323403750/5401361",
        "time_base": "1/90000",
        "start_pts": 0,
        "start_time": "0.000000",
        "duration_ts": 43210888,
        "duration": "480.120978",
        "bit_rate": "40292632",
        "bits_per_raw_sample": "8",
        "nb_frames": "28747",
        "extradata_size": 42,
        "disposition": {
          "default": 1,
          "dub": 0,
          "original": 0,
          "comment": 0,
          "lyrics": 0,
          "karaoke": 0,
          "forced": 0,
          "hearing_impaired": 0,
          "visual_impaired": 0,
          "clean_effects": 0,
          "attached_pic": 0,
          "timed_thumbnails": 0,
          "captions": 0,
          "descriptions": 0,
          "metadata": 0,
          "dependent": 0,
          "still_image": 0
        },
        "tags": {
          "creation_time": "2022-09-28T15:07:43.000000Z",
          "language": "und",
          "handler_name": "VideoHandle",
          "vendor_id": "[0][0][0][0]"
        }
      },
      {
        "index": 1,
        "codec_name": "aac",
        "codec_long_name": "AAC (Advanced Audio Coding)",
        "profile": "LC",
        "codec_type": "audio",
        "codec_tag_string": "mp4a",
        "codec_tag": "0x6134706d",
        "sample_fmt": "fltp",
        "sample_rate": "48000",
        "channels": 2,
        "channel_layout": "stereo",
        "bits_per_sample": 0,
        "id": "0x2",
        "r_frame_rate": "0/0",
        "avg_frame_rate": "0/0",
        "time_base": "1/48000",
        "start_pts": 0,
        "start_time": "0.000000",
        "duration_ts": 23045319,
        "duration": "480.110813",
        "bit_rate": "195525",
        "nb_frames": "22505",
        "extradata_size": 2,
        "disposition": {
          "default": 1,
          "dub": 0,
          "original": 0,
          "comment": 0,
          "lyrics": 0,
          "karaoke": 0,
          "forced": 0,
          "hearing_impaired": 0,
          "visual_impaired": 0,
          "clean_effects": 0,
          "attached_pic": 0,
          "timed_thumbnails": 0,
          "captions": 0,
          "descriptions": 0,
          "metadata": 0,
          "dependent": 0,
          "still_image": 0
        },
        "tags": {
          "creation_time": "2022-09-28T15:07:43.000000Z",
          "language": "und",
          "handler_name": "SoundHandle",
          "vendor_id": "[0][0][0][0]"
        }
      }
    ],
    "copyStreamIdsByFile": {
      "E:\\Videos\\R6 Clips\\Desktop\\Desktop 2022.09.29 - 01.06.28.02.DVR.mp4": {
        "0": true,
        "1": true
      }
    },
    "cutSegments": [
      {
        "start": 459.1549793103448,
        "end": 477.82244
      }
    ],
    "mainFileFormatData": {
      "filename": "E:\\Videos\\R6 Clips\\Desktop\\Desktop 2022.09.29 - 01.06.28.02.DVR.mp4",
      "nb_streams": 2,
      "nb_programs": 0,
      "format_name": "mov,mp4,m4a,3gp,3g2,mj2",
      "format_long_name": "QuickTime / MOV",
      "start_time": "0.000000",
      "duration": "480.126000",
      "size": "2430328964",
      "bit_rate": "40494852",
      "probe_score": 100,
      "tags": {
        "major_brand": "mp42",
        "minor_version": "0",
        "compatible_brands": "isommp42",
        "creation_time": "2022-09-28T15:07:43.000000Z",
        "date": "2022"
      }
    },
    "rotation": 360,
    "shortestFlag": false,
    "effectiveExportMode": "merge"
  },
  "platform": "win32",
  "version": "3.47.1"
}

### Expected behavior

After pressing the export button, the clipped video should just go to the output folder. 

### Actual behavior

Described in steps to reproduce box

### Share log

Not exactly sure as to what to include here, but this exists

![image](https://user-images.githubusercontent.com/117027914/198887384-5c00588b-8d49-4737-b99e-ba34fa87e099.png)

**Comment by mifi** [comment]
Hi! This is very strange. ffmpeg is just saying "Invalid argument". Did you try other output formats like mov, mkv? Alternatively if you can share a file that reproduces the problem i can try to look into it. If private you can send link to finstaden@gmail.com

**Comment by PinkDucki** [comment]
Hey, thanks for the reply. I have tried a whole variety of videos and the same issue occurs. This is the same with changing the output destination and output format. I also tried restarting my pc and nothing changes. I can send you any video from my pc if you would like as the issue occurs with any of them. 

Also something I should have, but forgot to mention in the post is that I had been using the program for about a month and it worked completely fine, but about a day ago I got another SSD added into my pc, and replaced my m.2 drive with my os on it for another, bigger drive. After these changes is when the exporting stopped working so looking back on it, that seems like it could be the problem?
